albatross/core/model_adapter.h

```diff
@@ -141,7 +141,7 @@ class AdaptedRegressionModel
     GenericModelFunctions<FeatureType, FitType> funcs;
 
     decltype(funcs.fitter) fitter =
-        [&](const std::vector<FeatureType> &features,
+        [=](const std::vector<FeatureType> &features,
```

Comment:
Yes good catch.

Reply:
I would also propose that if you only intend to capture one or two variables, it could even be better to name them explicitly. Doesn't really make a difference, but it can communicate intent sometimes.

Reply:
Good call, I've updated it to be explicit.